dis_snek/models/discord/guild.py

```diff
@@ -889,6 +889,48 @@ async def create_text_channel(
             reason=reason,
         )
 
+    async def create_forum_channel(
```

Comment:
The argument signature should be amended to include a `template` option. Easier to add it now as part of the `Create Channel` endpoint payload instead of delaying